From 682a9eabc71c2e33ebbbba2e75d0ba7caa08c7c8 Mon Sep 17 00:00:00 2001
From: Bernd Kuhls <bernd.kuhls@t-online.de>
Date: Mon, 8 May 2023 20:03:22 +0200
Subject: [PATCH] Don't force -fstack-protector

This allows the environment to decide whether SSP should be used or
not, for example to support toolchains that don't have SSP support.

Upstream: Not applicable

Signed-off-by: Bernd Kuhls <bernd.kuhls@t-online.de>
---
 cmake/CompileOptions.cmake | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/cmake/CompileOptions.cmake b/cmake/CompileOptions.cmake
index 9923f1c..5b24622 100644
--- a/cmake/CompileOptions.cmake
+++ b/cmake/CompileOptions.cmake
@@ -54,9 +54,6 @@ else()
       add_definitions("-D_FORTIFY_SOURCE=2")
     endif()
   endif()
-  if(NOT MINGW)
-    add_compile_options("-fstack-protector-strong")
-  endif()
   if(NOT MINGW)
     add_link_options("-Wl,-z,relro,-z,now,-z,noexecstack")
   endif()
-- 
2.39.2

